conf/messages

```diff
@@ -152,6 +152,9 @@ home.overdue.message.dunningLock.false                          = You have overd
 home.updates.overdue.payments                                   = {0} OVERDUE PAYMENTS
 home.updates.overdue.updates                                    = {0} OVERDUE UPDATES
 
+## Opt-Out ##
```

Comment:
Will having the opt-out messages separate from the main messages cause a problem when we eventually remove the feature switch? Might we end up leaving some old messages lying around? Would it be better to put these next to the messages they're replacing?

Reply:
Done